modules/datadog.nix

```diff
@@ -0,0 +1,13 @@
+{ config, pkgs, lib, ... }:
+
+with (import ./../lib.nix);
+
+{
+  config = {
+    services.dd-agent = {
```

Comment:
The name comes from NixOS upstream: https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/monitoring/dd-agent.nix

I agree abbreviations are confusing, we have a way to rename module attributes, but I'd leave this out of scope for this issue.